access_bam_qc/access_bam_qc.cwl

```diff
@@ -0,0 +1,957 @@
+class: Workflow
+cwlVersion: v1.0
+id: access_bam_qc
+label: access_bam_qc
+$namespaces:
+  s: 'https://schema.org/'
+  sbg: 'https://www.sevenbridges.com/'
+inputs:
+  - id: reference
+    type: File
+    secondaryFiles:
+      - ^.fasta.fai
+      - ^.dict
+    'sbg:x': -1367.075439453125
+    'sbg:y': -170.63369750976562
+  - id: duplex_bam
+    type: 'File[]'
+    label: duplex_bam
+    secondaryFiles:
+      - ^.bai
+    'sbg:x': -808.21337890625
+    'sbg:y': -258.8006591796875
+  - id: collapsed_bam
+    type: 'File[]'
+    label: collapsed_bam
+    secondaryFiles:
+      - ^.bai
+    'sbg:x': -810.0906372070312
+    'sbg:y': -123.79762268066406
+  - id: group_reads_by_umi_bam
+    type: 'File[]'
+    label: group_reads_by_umi_bam
+    doc: Input BAM file generated by GroupReadByUmi.
+    'sbg:x': -811.8580322265625
+    'sbg:y': 27.081497192382812
+  - id: uncollapsed_bam
+    type: 'File[]'
+    label: uncollapsed_bam
+    secondaryFiles:
+      - ^.bai
+    'sbg:x': -810.2417602539062
+    'sbg:y': 164.8699493408203
+  - id: uncollapsed_bam_base_recal
+    type: 'File[]'
+    label: uncollapsed_bam_base_recal
+    doc: An aligned SAM or BAM file.  Required.
+    secondaryFiles:
+      - ^.bai
+    'sbg:x': -813.2417602539062
+    'sbg:y': 310.27471923828125
+  - id: pool_b_target_intervals
+    type: File
+    label: pool_b_target_intervals
+    'sbg:x': -1362.9149169921875
+    'sbg:y': 304.2558288574219
+  - id: pool_b_bait_intervals
+    type: File
+    label: pool_b_bait_intervals
+    'sbg:x': -1362.1119384765625
+    'sbg:y': 466.5614929199219
+  - id: pool_a_target_intervals
+    type: File
+    label: pool_a_target_intervals
+    'sbg:x': -1358.999755859375
+    'sbg:y': -23.60011863708496
+  - id: pool_a_bait_intervals
+    type: File
+    label: pool_a_bait_intervals
+    'sbg:x': -1366.56494140625
+    'sbg:y': 126.96497344970703
+  - id: biometrics_bed_file
```

Comment:
Can this be the same as one of the other bed files?

Reply:
yes, i just need to modify biometrics so that it will filter out regions that are not Y chromosome. i'll so that now